README.md

```diff
@@ -93,21 +93,21 @@ A semi-populated example files directory might then contain:
     └── products/
         ├── universalforwarder/
         │   └── releases/
-        |       └── 7.0.0/
+        |       └── 9.2.0/
         |           ├── linux/
-        |           |   ├── splunkforwarder-7.0.0-c8a78efdd40f-linux-2.6-amd64.deb
-        |           |   ├── splunkforwarder-7.0.0-c8a78efdd40f-linux-2.6-intel.deb
-        |           |   └── splunkforwarder-7.0.0-c8a78efdd40f-linux-2.6-x86_64.rpm
+        |           |   ├── splunkforwarder-9.2.0-1fff88043d5f-linux-2.6-amd64.deb
+        |           |   ├── splunkforwarder-9.2.0-1fff88043d5f-linux-2.6-intel.deb
```

Comment:
2.6, does that refer to the kernel version? Is that still so old?

Reply:
I guess it does? 😂  
But it's just in the filename, with the RPMs they suddenly changed the URL and removed the `-linux-2.6-` thing.. (See https://github.com/voxpupuli/puppet-splunk/pull/350/commits/61285e23515ee7356be15d03046ccffcec85bae7)